waku/waku.md

```diff
@@ -163,8 +164,7 @@ packet-format = "[" packet-code packet-format "]"
 
 required-packet = 0 status /
                   1 messages /
-		  2 pow-requirement /
-		  3 bloom-filter
+		  2 status-update /
```

Comment:
We want something similar (or the same?) as this in ABNF:

```
status-options = "["
  [ pow-requirement-key pow-requirement ]
  [ bloom-filter-key bloom-filter ]
  [ light-node-key light-node ]
  [ confirmations-enabled-key confirmations-enabled ]
  [ rate-limits-key rate-limits ]
  [ topic-interest-key topic-interest ]
"]"
````

Reply:
E.g.

`status-update = status-options` (assuming that is accurate)

Reply:
updated